src/shapes/Textbox.ts

```diff
@@ -306,6 +310,58 @@ export class Textbox extends IText {
     return value.split(this._wordJoiners);
   }
 
+  measureLineForWrapping(line: string, lineIndex: number) {
+    const splitByGrapheme = this.splitByGrapheme,
```

Comment:
is this assignment really necessary as opposed to just using `this.splitByGrapheme`?

Reply:
agreed, it is a left over

Reply:
76eef16e2